cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -26,6 +26,15 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql;
 
+CREATE OR REPLACE FUNCTION set_bbox_to_envelope_of_geom()
```

Comment:
Could you just add a comment explaining this here

Reply:
Done!